components/PublicationDate/src/index.scss

```diff
@@ -1,26 +1,27 @@
 .denhaag-publication-date {
   color: var(--denhaag-publication-date-color);
+  display: flex;
+  flex-direction: var(--denhaag-publication-direction, column);
   font-family: var(--denhaag-publication-date-font-family);
   font-weight: var(--denhaag-publication-date-font-weight);
   font-size: var(--denhaag-publication-date-font-size);
+  padding-block-start: var(--denhaag-publication-date-padding);
+  padding-block-end: calc(var(--denhaag-publication-date-padding) * 2);
+  border-block-start: var(--denhaag-publication-date-border-width) var(--denhaag-publication-date-border-style)
+    var(--denhaag-publication-date-border-color);
 }
 
-.denhaag-publication-date > span {
-  display: block;
-  line-height: var(--denhaag-publication-date-line-height);
-  white-space: nowrap;
-}
+@media (min-width: 640px) {
+  .denhaag-publication-date {
+    --denhaag-publication-direction: row;
 
-@media (min-width: 480px) {
-  .denhaag-publication-date > span {
-    display: inline;
+    gap: var(--denhaag-publication-date-gap);
+    line-height: 1; // otherwise the vertical-line will be a lot higher.
   }
 
-  .denhaag-publication > span + span {
-    border-inline-end-color: var(--denhaag-publication-date-border-color);
-    border-inline-end-style: var(--denhaag-publication-date-border-style);
-    border-inline-end-width: var(--denhaag-publication-date-border-width);
-    margin-inline-end: var(--denhaag-publication-date-margin);
-    padding-inline-end: var(--denhaag-publication-date-padding);
+  .denhaag-publication-date > span + span {
```

Comment:
> As I'm looking at this component I think it could be more generic and better reusable in other applications if we didn't make it as publication date specific.
> 
> I made a PR based on this one with the changes I propose before merging. If you agree we can merge mine into yours or merge it into main instead. #1038
> 
> This PR also includes a fix for `default` and `--horizontal` styling that can be used in the website implementation like so:
> 
> ```css
> @media (min-width: x) {
>   .denhaag-article-meta {
>     @extends .denhaag-article-meta--horizontal;
>   }
> }
> ```

The component is designed to be vertical on smaller devices and horizontal on larger devices. Therefor there's a media-query. We do rather not add a className by javascript or extend with extra css for such a small component. We're required to match the design as designed: https://www.figma.com/file/JpoY3waVoQGlLQzQXTL9nn/Design-System---Gemeente-Den-Haag?node-id=8515%3A28504 Therefor i would say making 2 seperate variants which do not check on media-queries will only cause bloating.